sdk/rust/oak/build.rs

```diff
@@ -17,12 +17,13 @@
 fn main() {
     oak_utils::compile_protos_with_options(
         &[
+            "../../../oak_services/proto/grpc_invocation.proto",
+            "../../../oak_services/proto/http_invocation.proto",
             "../../../oak_services/proto/storage_service.proto",
             "../../../oak_services/proto/roughtime_service.proto",
         ],
         &["../../.."],
         oak_utils::ProtoOptions {
-            derive_handle_visit: false,
```

Comment:
Is `handle_visit` not necessary here? What does it do?

Reply:
By removing this line the HandleVisit trait will be automatically derived for the protos during compilation (the default is true). This is required for the new protos to be sent and received using Sender and Receiver.